README.adoc

```diff
@@ -91,116 +91,127 @@ image::find-build-scans.png[Find build scans,{thumbnail}]
+
 === Configuration options
 
-The following sections list all available configuration options which can be set as environment variables.
+The following sections list all available configuration options which can be set via the configuration form.
 
-==== General environment variables
+==== Common configuration options
 
-`JENKINSGRADLEPLUGIN_GRADLE_ENTERPRISE_INJECTION` +
+`Enable auto-injection` +
 Globally enable auto-injection.
 
-`JENKINSGRADLEPLUGIN_GRADLE_ENTERPRISE_URL` +
+`Gradle Enterprise server url` +
 The URL of the Gradle Enterprise instance.
 
-`JENKINSGRADLEPLUGIN_GRADLE_ENTERPRISE_ALLOW_UNTRUSTED_SERVER` +
+`Allow untrusted server` +
 Whether to allow publishing to a server with a self-signed certificate.
-Defaults to `false`.
 
-`GRADLE_ENTERPRISE_ACCESS_KEY` +
-If your Gradle Enterprise server requires authentication for publishing build scans you might want to set the `GRADLE_ENTERPRISE_ACCESS_KEY` environment variable.
+`Gradle Enterprise access key` +
+The access key for authenticating with the Gradle Enterprise server.
+The access key is stored encrypted within the Jenkins instance. During the job execution, the access key is decrypted and set as `GRADLE_ENTERPRISE_ACCESS_KEY` environment variable.
 Refer to the https://docs.gradle.com/enterprise/gradle-plugin/#via_environment_variable[Gradle Enterprise Gradle plugin manual] and the https://docs.gradle.com/enterprise/maven-extension/#via_environment_variable[Gradle Enterprise Maven Extension manual] for more details about this configuration.
 
-==== Gradle environment variables
+==== Gradle specific configuration options
 
-`JENKINSGRADLEPLUGIN_GRADLE_ENTERPRISE_PLUGIN_VERSION` +
+`Gradle Enterprise Gradle plugin version` +
 Enables auto-injection for Gradle builds and defines which version of the https://docs.gradle.com/enterprise/gradle-plugin/[Gradle Enterprise Gradle plugin] to use.
 
-`JENKINSGRADLEPLUGIN_CCUD_PLUGIN_VERSION` +
+`Common Custom User Data Gradle plugin version` +
 Defines which version of the https://github.com/gradle/common-custom-user-data-gradle-plugin[Common Custom User Data Gradle plugin] to use.
 
-`JENKINSGRADLEPLUGIN_GRADLE_PLUGIN_REPOSITORY_URL` +
+`Gradle plugin repository url` +
 The URL of the repository to use to resolve the Gradle Enterprise Gradle plugin and the Common Custom User Data Gradle plugin.
 This is required if the Jenkins agents are not able to access the https://plugins.gradle.org[Gradle Plugin Portal].
 
-`JENKINSGRADLEPLUGIN_GRADLE_INJECTION_ENABLED_NODES` +
-Comma separated list of node labels on which the Gradle Enterprise Gradle plugin injection should be enabled.
+`Gradle auto-injection enabled nodes` +
+A list of node labels on which the Gradle Enterprise Gradle plugin injection should be enabled.
```

Comment:
```suggestion
A list of node labels on which the Gradle Enterprise Gradle plugin or Common Custom User Data Gradle plugin injection should be enabled.
```

Please adapt this on all 4 sections about node labels (Gradle and Maven)

Reply:
Done